lua/ide/workspaces/workspace_controller.lua

```diff
@@ -246,25 +246,42 @@ function WorkspaceController.new(config)
         -- issues, so do it only after "VimEnter" fires.
         local restore = libwin.restore_cur_win()
         local init_autocmd = nil
-        init_autocmd =  vim.api.nvim_create_autocmd({ "VimEnter" }, {
-            callback = function()
-                -- do not assign if we are git tool.
-                local buf_name = vim.api.nvim_buf_get_name(0)
-                if vim.fn.match(buf_name, ".git/*") >= 0 then
+        local function init_callback()
+            -- do not assign if we are git tool.
+            local buf_name = vim.api.nvim_buf_get_name(0)
```

Comment:
hey there, just to minimize the diff in this change, can we omit this? it servers no functional improvement, right?

Reply:
Yeah, I was using it multiple times before but now I’m not. I can change this back shortly

Reply:
Done!